### ISSUE TITLE
Question of C-Index

### ISSUE THREAD

**Issue by Hanminghao** [Innse/MOTCat#4]
Hello author, I would like to ask if you have ever encountered the problem that loss decreases in the training set and verification set, and C-Index increases in the training set but remains unchanged or decreases in the verification set. May I ask how it was resolved? Looking forward to your reply!

**Comment by Innse** [comment]
Yes. It would occasionally happen. The calculations of NLL loss and C-Index are different. So loss decreases cannot ensure C-Index increases. “C-Index increases in the training set but remains unchanged or decreases in the verification set.” seems to indicate overfitting. You can try to use some regularization techniques, such as tuning dropout. Hope it works for you.
